homeassistant/components/hue/config_flow.py

```diff
@@ -50,6 +52,13 @@ def _async_get_bridge(self, host: str, bridge_id: Optional[str] = None):
             bridge_id=bridge_id,
         )
 
+    def _update_existing_entry(self, bridge: aiohue.Bridge) -> None:
```

Comment:
Let's raise `AbortFlow` in here. Then all try…except code can go where it's called.